src/Components/Admin/AssetModelViewer/assetModelViewer.js

```diff
@@ -149,6 +162,8 @@ export default function AssetModelViewer({
                         value={name}
                         onChange={handleNameChange}
                         variant="outlined"
+                        error={!empty(nameError)}
```

Comment:
Oh interesting why do we have `empty`? Couldn't we just compare against `""`? Or am I missing something?

Reply:
Oh yup, we can do that instead. I just copied the validation logic from login.js which was using this for a slightly more complicated use case.

Reply:
We could, I think empty() is safer and also checks for null or other "empty"-type states. 

This covers the other states they detect.
https://www.npmjs.com/package/is-empty